common/src/main/scala/org/apache/spark/sql/comet/util/Utils.scala

```diff
@@ -223,9 +223,9 @@ object Utils extends CometTypeShim {
       writer.close()
 
       if (out.size() > 0) {
-        (batch.numRows(), cbbos.toChunkedByteBuffer)
+        (batch.numRows().toLong, cbbos.toChunkedByteBuffer)
```

Comment:
what was the warning for this change?

Reply:
Thanks @andy-hf-kwok so in fact the compiler made the implicit conversion to long?